packages/webex/src/meetings.js

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+// Note: this file is written using commonjs instead of import/export to
+// simplify consumption by those less familiar with the current state of
+// JavaScript modularization
+
+/* eslint camelcase: [0] */
+
+/* istanbul ignore else */
+if (!global._babelPolyfill) {
+  /* eslint global-require: [0] */
+  require('@babel/polyfill');
+}
+
+require('@webex/plugin-authorization');
+// explicitly load wdm, since we're relying on preDiscoveryServices and the
+// url interceptor
+require('@webex/plugin-logger');
+require('@webex/common');
+require('@webex/plugin-meetings');
+require('@webex/internal-plugin-device');
+require('@webex/internal-plugin-metrics');
+require('@webex/internal-plugin-support');
+require('@webex/internal-plugin-user');
+require('@webex/plugin-people');
+
+const merge = require('lodash/merge');
+const WebexCore = require('@webex/webex-core').default;
+
+const config = require('./config');
+
+// documentation.js puts hashes in relative urls, so need to specify full urls
+// here
+/**
+ * See {@link https://webex.github.io/webex-js-sdk/example/browsers|Browser Guide} and
+ * {@link https://webex.github.io/webex-js-sdk/example/servers|Server Guide}
+ * @see {@link /example/browsers|Browser Guide}
+ * @see {@link /example/servers|Server Guide}
+ * @class Webex
+ */
+const Webex = WebexCore.extend({
+  webex: true,
+  version: PACKAGE_VERSION,
+});
+
+/**
+ * Create a new {@link Webex} instance
+ *
+ * @example
+ * <caption>Create a new Webex instance configured for your OAuth client</caption>
+ * const webex = Webex.init({
+ *   config: {
+ *     credentials: {
+ *       authorizationString: `<AUTHORIZATION URL FROM DEVELOPER PORTAL>`
+ *     },
+ *   }
+ * });
+ *
+ * @example
+ * <caption>Create a new Webex instance configured for a Bot</caption>
+ * const webex = Webex.init({
+ *   credentials: `<BOT TOKEN FROM DEVELOPER PORTAL>`,
+ * });
+ *
```

Comment:
Do we really need all these comments in the code? Seems more like documentation to me.